components/match2/commons/match.lua

```diff
@@ -27,6 +27,7 @@ function Match.storeFromArgs(frame)
 end
 
 function Match.toEncodedJson(frame)
+	FeatureFlag.set('combined_opponent_input', true)
```

Comment:
What does this specific FeatureFlag do?

Reply:
It signals the opponent input templates (like Template:1Opponent) to not do any processing.